File: ex7/ex7.erl
-module(ex7).
-import(lists, [filter/2]).
-export([splitEvenOdds/1]).

splitEvenOdds([]) -> [];
splitEvenOdds(X) ->{lists:filter(fun (Y) -> Y rem 2 == 1 end, X), 
                    lists:filter(fun (Y) -> Y rem 2 == 0 end, X)}.

% c(ex7).
% ex7:splitEvenOdds([1,2,3,4,5,6,7]).

File: ex7 (Victor)/ex7.hs
-- isEven :: Int -> Bool
-- isEven n = n `mod` 2 == 0

splitEvenOdds :: [Int] -> [[Int]]
--splitEvenOdds xs = [filter (not . isEven) xs, filter isEven xs]
splitEvenOdds xs = [filter odd xs, filter even xs]

-- ghci ex7.hs
main = do 
   putStrLn "[[Impares],[Pares]]"  
   print(splitEvenOdds [1,2,3,4,5,6,7])